src/backend/distributed/master/master_node_protocol.c

```diff
@@ -217,7 +218,7 @@ master_get_table_ddl_events(PG_FUNCTION_ARGS)
 		oldContext = MemoryContextSwitchTo(functionContext->multi_call_memory_ctx);
 
 		/* allocate DDL statements, and then save position in DDL statements */
-		tableDDLEventList = GetTableDDLEvents(relationId);
+		tableDDLEventList = GetTableDDLEvents(relationId, includeSequenceDefaults);
```

Comment:
Should this be true? I guess we only use this from FetchRegularTable and if the defaults weren't in the original shard, then I suppose they won't be the copied shard.